app/src/main/java/com/dimelo/sampleapp/chats/TabBank.java

```diff
@@ -55,6 +56,8 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
     public void onViewCreated(final View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         FragmentManager childFragmentManager = getChildFragmentManager();
+        TextView versionName = view.findViewById(R.id.versionName);
+        versionName.setText("V."+Dimelo.getInstance().getVersionName());
```

Comment:
`Dimelo.getInstance().getVersionName()`

please use the same iOS name so we can rename it to `Dimelo.getInstance().getSdkVersion()`

Reply:
ok